src/components/AdditionalInfo/AdditionalInfo.jsx

```diff
@@ -3,21 +3,36 @@ import PropTypes from 'prop-types';
 import _ from 'lodash';
 import classNames from 'classnames';
 import ExpandingGroup from '../ExpandingGroup/ExpandingGroup';
+import dispatchAnalyticsEvent from '../../helpers/analytics';
 
 export default class AdditionalInfo extends React.Component {
   constructor(props) {
     super(props);
     this.expandedContentId = _.uniqueId('tooltip-');
     this.state = { open: false };
+
+    this.toggle = this.toggle.bind(this);
   }
 
-  toggle = () => {
+  toggle() {
```

Comment:
I think this change was before I made the linting updates, yeah? Are you still running into problems with the arrow function syntax?